version/version.go

```diff
@@ -1,7 +1,7 @@
 package version
 
 const Maj = "0"
-const Min = "8" // validator set changes, tmsp->abci, app persistence/recovery, BFT-liveness fix
```

Comment:
I've been keeping the changelog in the release notes on github for every release. Not opposed to also adding them all into a CHANGELOG.md. the comment here is just a rough reference if you're cruising by